Samples/V1.0Samples/LocalMediaSamples/PolicyRecordingBot/README.md

```diff
@@ -147,20 +147,32 @@ Step 3: Define Your Virtual Network
    
-Step 4: Create Your Cloud Service (Extended Support)
-      You can deploy your Azure extended service using various methods, such as the Azure portal, PowerShell, Azure Resource Manager templates, or Visual Studio.
-
-1. To create a Cloud Service (Extended Support) in Azure, follow these steps:
-   
-### Note: 
-
-   1. Configure Storage Account for Configuration Files.
-    * To store configuration files for your Azure extended service, you'll need to set up a storage account. Follow these steps to configure the storage account:(https://learn.microsoft.com/en-us/azure/storage/common/storage-account-create?tabs=azure-portal).
-
-   2. Package Your Cloud Service for Deployment
-    * Before you can create your Azure extended service, you need to package your cloud service application to include configuration files and dependencies.
-    * Right click PolicyRecordingBot, then click `Package...`.
+ Step 4: Deploy
+
+   1. Create Your Cloud Service (Extended Support)
+      1. Use the Azure portal to create a Cloud Service (Extended Support). 
+        Follow this guide: Create a Cloud Service (Extended Support).(https://learn.microsoft.com/en-us/azure/cloud-services-extended-support/deploy-portal)
+      2. Obtain Your Public IP DNS name:
+        After the service is created, obtain the "Public IP DNS name" from the Azure portal. This URL will serve as your DNS name and Common Name (CN) for further configurations (e.g. bot.contoso.com).
+        ![Public IP DNS name](Images/PublicIPDNSName.png).  
+
+   2. Update the app configs with values
+      1. Set up cloud service configuration
+        1. Open powershell, go to the folder that contains file `configure_cloud.ps1`. The file is in the `Samples` directory.
+        2. Run the powershell script with parameters:
+          ` .\configure_cloud.ps1 -p {path to project} -dns {your DNS name} -cn {your CN name, should be the same as your DNS name} -bid {your bot name} -aid {your bot app id} -as {your bot secret}`
+        
+         For example:
+        
+         `.\configure_cloud.ps1 -p .\V1.0Samples\LocalMediaSamples\PolicyRecordingBot\ -dns bot.contoso.com -cn bot.contoso.com -bid bot -aid 3853f935-2c6f-43d7-859d-6e8f83b519ae -as 123456!@#$%^`
 
+   3. Deploy to Cloud Service (Extended Support)
+     1. Configure Storage Account for Configuration Files.
+       * To store configuration files for your Azure extended service, you'll need to set up a storage account. Follow these steps to configure the storage account:(https://learn.microsoft.com/en-us/azure/storage/common/storage-account-create?tabs=azure-portal).
+     2. Package Your Cloud Service for Deployment
+       * Before you can create your Azure extended service, you need to package your cloud service application to include configuration files and dependencies.
+       * Right click PolicyRecordingBot, then click `Package...`.
+     
    Option 1: Upload to Azure Storage Account:
```

Comment:
If we are saying option1 then we might as well have option2 instead of Alternatively.